config/monitoring/metrics/stackdriver/100-stackdriver-serviceentry.yaml

```diff
@@ -0,0 +1,68 @@
+# Copyright 2018 The Knative Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+apiVersion: networking.istio.io/v1alpha3
```

Comment:
Can you please open an issue to replace these using ServiceEntry "*" when the feature is available in Istio?

Thanks

Reply:
https://github.com/knative/serving/issues/2367